ipie/utils/chunk_large_chol.py

```diff
@@ -3,22 +3,35 @@
 import numpy as np
 
 
-def split_cholesky(ham_filename, nmembers):
-    #
+def split_cholesky(ham_filename: str, nmembers: int, verbose=True):
+    """
+    This function calculates the splits and displacements needed to distribute the
+    Cholesky vectors among the members and  splits the Cholesky decomposed Hamiltonian
+    vectors stored in an HDF5 file among a given number of members
+    (e.g., GPU cards to distribute total cholesky)
+
+    Parameters
+    ----------
+    ham_filename : str
+        The filename of the HDF5 file containing the total Cholesky (naux, nbas, nbas)
+    nmembers : int
+        The number of members among which the Cholesky vectors will be distributed.
+    """
     with h5py.File(ham_filename, "r") as source_file:
         dataset = source_file["LXmn"][()]  # pylint: disable=no-member
```

Comment:
Why is this pylint disable needed here?

Reply:
I just removed it. I have to explicitly add np.array(source_file["LXmn"][()]) so that pylint can recognize